Makefile

```diff
@@ -15,9 +15,6 @@ ${DIR}/boogie/Binaries/Boogie.exe:
 refman:
 	make -C ${DIR}/docs/DafnyRef
 
-refman-release:
```

Comment:
Why are you confident about deleting this?

Reply:
It's not used

Reply:
Well, I guess that's not right.  Let me revert this bit.